<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorFindReplaceWidget</class>
 <widget class="QWidget" name="EditorFindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>240</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Find:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <layout class="QHBoxLayout" name="replaceLayout">
     <item>
      <widget class="QToolButton" name="replaceButton">
       <property name="toolTip">
        <string>Replace</string>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="replaceAndFindButton">
       <property name="toolTip">
        <string>Replace &amp; Find</string>
       </property>
       <property name="text">
        <string>Replace &amp;&amp; Find</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="replaceAllButton">
       <property name="toolTip">
        <string>Replace All</string>
       </property>
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="replaceLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Replace with:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="findEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="replaceEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="findLayout">
     <item>
      <widget class="QToolButton" name="findPreviousButton">
       <property name="toolTip">
        <string>Find Previous</string>
       </property>
       <property name="icon">
        <iconset resource="../res/Editor_ui.qrc">
         <normaloff>:/oxygen/actions/go-previous.png</normaloff>:/oxygen/actions/go-previous.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="findNextButton">
       <property name="toolTip">
        <string>Find Next</string>
       </property>
       <property name="icon">
        <iconset resource="../res/Editor_ui.qrc">
         <normaloff>:/oxygen/actions/go-next.png</normaloff>:/oxygen/actions/go-next.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="findSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>findEdit</tabstop>
  <tabstop>replaceEdit</tabstop>
  <tabstop>findPreviousButton</tabstop>
  <tabstop>findNextButton</tabstop>
  <tabstop>replaceButton</tabstop>
  <tabstop>replaceAndFindButton</tabstop>
  <tabstop>replaceAllButton</tabstop>
 </tabstops>
 <resources>
  <include location="../res/Editor_ui.qrc"/>
 </resources>
 <connections/>
</ui>
